fern/docs/pages/snippet-templates.mdx

```diff
@@ -0,0 +1,13 @@
+import CodeSnippet from "../snippets/code-snippet.mdx";
+
+<CodeSnippet title="Example 1" components={props.components}>
```

Comment:
at the moment there's a bug where `components={props.components}` need to be passed in explicitly. We'll need to fix this on our side so it's handled automatically.